com.avaloq.tools.ddk.xtext.format/model/generated/Format.ecore

```diff
@@ -3,7 +3,7 @@
     xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore" name="format" nsURI="http://www.avaloq.com/tools/ddk/xtext/format/Format"
     nsPrefix="format">
   <eClassifiers xsi:type="ecore:EClass" name="FormatConfiguration">
-    <eStructuralFeatures xsi:type="ecore:EReference" name="targetGrammar" eType="ecore:EClass platform:/plugin/org.eclipse.xtext/org/eclipse/xtext/Xtext.ecore#//Grammar"/>
+    <eStructuralFeatures xsi:type="ecore:EReference" name="targetGrammar" eType="ecore:EClass platform:/resource/org.eclipse.xtext/org/eclipse/xtext/Xtext.ecore#//Grammar"/>

```

Comment:
this changes are not correct. Could we revert the file?

Reply:
reverted. thanks!